src/intelligence_layer/connectors/argilla/argilla_wrapper_client.py

```diff
@@ -37,24 +33,24 @@ def __init__(
                 logging.WARNING
             )
 
-        rg.init(
+        self.client = rg.Argilla(
             api_url=api_url if api_url is not None else os.getenv("ARGILLA_API_URL"),
             api_key=api_key if api_key is not None else os.getenv("ARGILLA_API_KEY"),
         )
 
     def create_dataset(
         self,
-        workspace_id: str,
+        workspace_name: str,
         dataset_name: str,
-        fields: Sequence[AllowedFieldTypes],
-        questions: Sequence[AllowedQuestionTypes],
+        fields: Sequence[rg.TextField],
```

Comment:
Is everything a text field?

Reply:
Interestingly yes, was skeptical at first myself 😄.